superset/views/geocoding.py

```diff
@@ -77,11 +77,14 @@ def _get_editable_tables(self):
         for database in (
             db.session.query(models.Database).filter_by(allow_dml=True).all()
         ):
-            # TODO loop through all schemas for postgres databases
             for table in (
                 db.session.query(SqlaTable).filter_by(database_id=database.id).all()
             ):
-                tables.append(models.TableDto(table.id, table.name, table.database_id))
+                tables.append(
+                    models.TableDto(
+                        table.id, table.table_name, table.schema, table.database_id
+                    )
+                )
```

Comment:
Does this loop through all schemas as well? Only found this in a quick search: https://stackoverflow.com/questions/6473925/sqlalchemy-getting-a-list-of-tables
which is a different approach. Should we write a test for this or would this be unnecessary?

Reply:
yes, i found this because of the bug with postgres. i have a postgres database with schema name public and a table in it. this method returns all tables in the database, with schema and without